### PR TITLE
DDF-4982 Improve Search Form and Map Action UX

### PR DESCRIPTION
**Pull request by Bdthomson**
#### What does this PR do?
- Refactors search forms to reduce duplicated code
- Fixes UX issues with search and result forms
	- Switching tabs no longer refreshes the entire view (including menu bar)
	- Proper messages are shown when no forms are found.
- Fixes UX issues with map actions
	- Actions acting on selected results now identify as such

#### Who is reviewing it? 
@djblue 
@samuelechu 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@adimka
@bdeining

#### How should this be tested?
Map Actions
1. Interact with the map and click map actions to ensure they still work as expected.

Search Forms
1. Download [forms.zip](https://github.com/codice/ddf/files/3308001/forms.zip), unzip, and place contents in `$DDF_HOME/etc/forms/`.
2. In Karaf console run `forms:load` to load system templates.
3. Test
a. Creating a search form
b. Sharing a search form (and opening on another user)
c. Opening system search forms
d. Querying with search forms

#### What are the relevant tickets?
Fixes: #4982 

##### Search Forms
*Before*
DDF-4982
*After*
SearchFormsAfter

##### Map Actions
*Before*
image

*After*
image

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by willwill96** [comment]
✏️ It looks like the text under My/Shared/System Search Forms isn't centered in the After gif

File: ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form-presentation.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from '../../react-component/styles/styled-components'
+
+export const TabMessage = styled.div`
+  text-align: center;
+  font-size: 20px;
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form-presentation.js]:
```
${props => props.theme.largeFontSize}
```

Comment (Bdthomson) [reply ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form-presentation.js]: Done

**Comment by Bdthomson** [comment]
build now

**Comment by bellcc** [comment]
**Hero Successful** 🎉 

- Map actions work as expected. 
- Loaded some search forms into DDF.
- Verified that querying and sharing them works as expected.

**Pull request merged**